pkg/scheduler/scheduler.go

```diff
@@ -4,18 +4,25 @@ import (
 	"context"
 	"errors"
 	"flag"
+	"net/http"
 	"sync"
 	"time"
 
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
+	otgrpc "github.com/opentracing-contrib/go-grpc"
 	"github.com/opentracing/opentracing-go"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promauto"
 	"github.com/weaveworks/common/httpgrpc"
+	"github.com/weaveworks/common/middleware"
+	"github.com/weaveworks/common/user"
 	"go.uber.org/atomic"
+	"google.golang.org/grpc"
 
+	"github.com/cortexproject/cortex/pkg/querier/frontend2"
```

Comment:
In other places I named this import `querier_frontend`. Do you mind using the same naming? We'll soon refactor it too, but in the meanwhile we keep consistency